lib/init-transforms/entry.js

```diff
@@ -0,0 +1,6 @@
+const run = (config, answer) => {
+	config.entry('index').add(answer).end();
```

Comment:
Sometimes it returned an object which was one level up starting with a `parentPath`
. When I added `.end()` it worked consistently everywhere.